README.md

```diff
@@ -47,11 +47,12 @@ RUN_INTEGRATION_TEST=all make test
 
 This repository is integrated with GitHub Actions.
 
-![Check changes in the Main branch](https://github.com/napptive/catalog-manager/workflows/Check%20changes%20in%20the%20Main%20branch/badge.svg)
+<a href="https://github.com/hasnainmakada-99/catalog-manager.git">![Check changes in the Main branch](https://github.com/napptive/catalog-manager/workflows/Check%20changes%20in%20the%20Main%20branch/badge.svg)
```

Comment:
Please remove this change as the badge is not intended to be a link, just to show the status. Moreover, the links should point to the main repo, not to a fork.